app/templates/components/ui-kit/box.hbs

```diff
@@ -0,0 +1,6 @@
+{{#let (element this.tag) as |Tag|}}
+  <Tag
```

Comment:
This is neat! 👍

Reply:
I agree, thanks for suggesting this addon :smile: